setup.py

```diff
@@ -5,56 +5,60 @@
 setuptools.setup(
     name=__name__,
     version=__version__,
-    description='Object browser for OS_swift API',
+    description="Object browser for OS_swift API",
     author=__author__,
     author_email="sampsa.penna@csc.fi",
     project_urls={
-        'Source': 'https://github.com/CSCfi/swift-browser-ui',
+        "Source": "https://github.com/CSCfi/swift-browser-ui",
     },
-    license='MIT',
+    license="MIT",
     install_requires=[
-        'aiohttp',
-        'python-swiftclient',
-        'cryptography',
-        'keystoneauth1',
-        'click',
-        'gunicorn',
-        'uvloop',
-        'certifi'
+        "aiohttp",
```

Comment:
can we pin versions on these as well or we can follow metadata-submitter and load `requirements.txt` file https://github.com/CSCfi/metadata-submitter/blob/master/setup.py